app/src/main/java/com/vero/tasky/agenda/domain/model/Day.kt

```diff
@@ -0,0 +1,11 @@
+package com.vero.tasky.agenda.domain.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import java.time.LocalDate
+
+@Parcelize
+data class Day(
+    val date: LocalDate,
+    val isSelected: Boolean
+) : Parcelable
```

Comment:
This is information which is only for the presentation layer (`isSelected` also reveals that, since that info is just needed to display something), therefore this should go in presentation